domain/data/slack/user.py

```
@@ -1,14 +1,25 @@
 """
-Slack user model.
+Slack user model with multi-store capabilities.
 """
 
 from sqlalchemy import Column, String, DateTime, JSON, Boolean
 from sqlalchemy.sql import func
 from .base import SlackBase
+from weave.bin.modules.annotations import neo4j_node, elasticsearch_index, neo4j_relationship
```

Ruff (0.11.9)

`weave.bin.modules.annotations.neo4j_relationship` imported but unused

Remove unused import: `weave.bin.modules.annotations.neo4j_relationship`

(F401)